crates/pop-cli/src/commands/call/parachain.rs

```diff
@@ -285,6 +316,9 @@ struct CallParachain {
 	suri: String,
 	/// Whether to automatically sign and submit the extrinsic without prompting for confirmation.
 	skip_confirm: bool,
+	/// Whether to authenticate with the sudo key and dispatch the function call with `Root`
```

Comment:
```suggestion
	/// Whether to dispatch the function call with `Root` origin.
```
Internal struct so a simpler comment is fine imo.

Reply:
Improved in https://github.com/r0gue-io/pop-cli/pull/349/commits/620252fefff47e47f786f3ba4f5dbbfeba00bd74